tensorflow/python/keras/utils/vis_utils.py

```diff
@@ -63,6 +64,34 @@ def add_edge(dot, src, dst):
     dot.add_edge(pydot.Edge(src, dst))
 
 
+def is_regex_match(pat, text):
```

Comment:
This function is not necessary. You should directly use re.match(pattern, layer_name), which is more readable.

Reply:
done